### PR TITLE
Store PDF documents in db table

### PR DESCRIPTION
**Pull request by danielnaab**
Finishing up #222, this PR:

- Adds PDF document storage to the database, removing it from the form definition
- Adds `initializeForm` service for creating new forms
- Gets attachment download working
- Adds validation of form blueprint at the service and database levels

### CODE REVIEW

File: packages/forms/src/services/initialize-form.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    expect(result).toEqual({
+      success: true,
+      data: {
+        timestamp: expect.any(String),
+        id: expect.any(String),
+      },
+    });
+  });
+
+  it('initializes successfuly with document when user is logged in', async () => {
```
Comment (ethangardner) [inline packages/forms/src/services/initialize-form.test.ts:38]: Typo in "successfully"

File: packages/server/src/lib/api-client.ts [hunk 2]
```diff
@@ -16,6 +17,7 @@ export class FormServiceClient implements FormService {
   constructor(private ctx: FormServiceClientContext) {}
 
   async addForm(form: Blueprint) {
+    console.log('add form');
```
Comment (ethangardner) [inline packages/server/src/lib/api-client.ts:20]: Can we remove or are there cases we still need to log to see if this method is called?

File: packages/design/src/FormManager/FormList/store.ts [hunk 3]
```diff
@@ -51,7 +50,16 @@ export const createFormListSlice =
...
       await builder.addDocument(fileDetails);
-      const result = await context.formService.addForm(builder.form);
+      const result = await context.formService.initializeForm({
+        summary: {
+          title: fileDetails.name,
+          description: '',
+        },
+        document: {
+          fileName: fileDetails.name,
+          data: await uint8ArrayToBase64(fileDetails.data),
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply packages/design/src/FormManager/FormList/store.ts:60]: Yes, right now it's all pretty SPA-like. It'd probably be pretty difficult to progressively enhance the builder UI, but maybe there's a tidy way to do it?

File: packages/forms/src/builder/index.ts [hunk 4]
```diff
@@ -23,6 +25,64 @@ import {
...
+export const parseForm = (config: FormConfig, obj: any): Result<Blueprint> => {
+  const formSchema = createFormSchema(config);
+  const result = formSchema.safeParse(obj);
+  if (result.error) {
+    return failure(result.error.message);
+  }
+  return success(result.data);
+};
+
+export const parseFormString = (
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply packages/forms/src/builder/index.ts:80]: Thanks for catching this. I had originally intended to use this when creating the object after reading from the DB, but found the dependency injection as implemented in the `createService` factory didn't support passing down what I needed, and got distracted and didn't make the necessary changes over there. I'll revisit that.

**Review by ethangardner — APPROVED** [review]
Thanks for addressing the items from the previous code review. I think this looks great!

**Pull request merged**